Source/Dafny/Resolver.cs

```diff
@@ -14427,6 +14427,7 @@ public class ResolveOpts
           Expression receiver;
           if (!member.IsStatic) {
             receiver = expr.Lhs;
+            AddAssignableConstraint(expr.tok, tentativeReceiverType, receiver.Type, "receiver type does not have a member named " + name);
```

Comment:
Add name of receiver type to error message. You can you do simply as follows:

            AddAssignableConstraint(expr.tok, tentativeReceiverType, receiver.Type, "receiver type ({1}) does not have a member named " + name);

Reply:
Done